website/docs/r/aviatrix_remote_syslog.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "Settings"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_remote_syslog"
+description: |-
+  Enables and disables remote syslog
+---
+
+# aviatrix_remote_syslog
+
+The **aviatrix_remote_syslog** resource allows the enabling and disabling of remote syslog.
+
+## Example Usage
+
+```hcl
+# Enable remote syslog without TLS
+resource "aviatrix_remote_syslog" "test_remote_syslog" {
+  index    = 0
+  server   = "1.2.3.4"
+  port     = 10
+  protocol = "TCP"
+}
+```
+
+```hcl
+# Enable remote syslog with TLS
+resource "aviatrix_remote_syslog" "test_remote_syslog" {
+  index              = 0
+  server             = "1.2.3.4"
+  port               = 10
+  protocol           = "TCP"
+  ca_certificate     = "/path/to/ca.pem"
+  public_certificate = "/path/to/server.pem"
+  private_key        = "/path/to/client.pem"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+* `index` (Optional) Profile index. An index from 0 to 9 is supported. 0 by default.
+* `server` (Required) Server IP.
+* `port` (Required) Port number.
+* `protocol` (Optional) TCP or UDP. TCP by default.
+
+### Optional
+* `ca_certificate` (Optional) Certificate Authority (CA) certificate. `ca_certificate`, `public_certificate` and `private_key` must be used together.
```

Comment:
It needs to be more clear that these are file paths not the content of the certificates. Could we add a suffix like `_file_path` to the certificate attribute names. Also make sure the documentation is clear that this is a file path.